src/devices/src/virtio/fs/linux/passthrough.rs

```diff
@@ -527,6 +527,8 @@ impl PassthroughFs {
             debug_assert!(namelen <= back.len(), "back is smaller than `namelen`");
 
             let name = &back[..namelen];
+            let term = name.iter().position(|&a| a == 0).expect("LinuxDirent64 name not NUL-terminated");
+            let name = &name[..term];
```

Comment:
Shouldn't be just `name` without `let`, while the variable exists already?

Reply:
https://users.rust-lang.org/t/mut-vs-redefinition/51098/3

Reply:
That would not work since `name` is not mutable, but there's also no reason to make it mutable (that's more for loops and things like that). As far as I know rebinding names like this is considered good practice.

Reply:
Fine with me 👍